spec/controllers/stories_bulk_update_controller_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'rails_helper'
+
+describe StoriesBulkUpdateController do
+  describe '#create' do
+    def catch_column(model, arg)
+      model.map do |record|
+        record.reload[arg.to_sym]
+      end
+    end
+
+    def response_body(status)
```

Comment:
What do you think about renaming this argument to `response_field` instead of `status`? It's not exactly the status, right?

Reply:
Sure, I didn't find a good name. Thanks